examples/JoystickControl/JoystickControl.ino

```diff
@@ -0,0 +1,56 @@
+#include 
+Servo servo1;     // 2 Servos Initialized 
+Servo servo2;
+int x_key = A1;     //Horizontal pin for Joystick                                         
+int y_key = A0;     //Vertical pin for Joystick                                         
+int x_pos;
+int y_pos;
+int servo1_pin = 8;   //Signal Pins for Servo
+int servo2_pin = 9;  
+int initial_position = 90;    //Initially Servos are moved to 90
+int initial_position1 = 90;
```

Comment:
These are inaccurate variable names. You are using these variables to store the position, not only the initial position. Also, `initial_position` vs `initial_position1` is not a very clear distinction. If you were to number the variable, `initial_position1` and `initial_position2` would be more sensible.

When you start numbering your variables it usually means you would be better off using an array:
```c++
servoPosition[] = {90, 90};
```

Reply:
Changed to initial_Position1 and initial_Position2 to avoid the complexity of arrays